tests/output/ConsoleInteractionTest.py

```diff
@@ -439,8 +428,9 @@ def test_print_result_no_input(self):
                                  """
 Project wide:
 |    | [NORMAL] origin:
-|    | message
-""")
+|    | {}""".format(ConsoleInteractionTest.highlightoutput(
```

Comment:
if you want you can use `self` instead of `ConsoleInteractionTest` to shorten the stmt^^